modules/swagger-codegen/src/main/resources/aspnet5/model.mustache

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Linq;
+using System.IO;
+using System.Text;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+{{#models}}
+{{#model}}
+namespace {{packageName}}.Models
+{
+    /// <summary>
+    /// {{description}}
+    /// </summary>
+    public class {{classname}} : {{#parent}}{{{parent}}}, {{/parent}} IEquatable<{{classname}}>
```

Comment:
I can add that. After we recently regenerated the C# client at work, I was thinking about how nice it would be if this was had the partial keyword.

I'll look at the client code, but are there any plans or best practices for partial methods? Or just the class for now?